### PR TITLE
Remove title from preview

### PR DESCRIPTION
**Pull request by sweatC**

### CODE REVIEW

**Comment by sweatC** [comment]
Without company title:
image

**Comment by sweatC** [comment]
@pftg check: http://jetthoughts-pr396.surge.sh/ on  https://metatags.io/
image

**Comment by pftg** [comment]
@sweatC https://github.com/jetthoughts/jetthoughts.github.io/pull/396#issuecomment-485453094 have not been fixed, what are you going to do with it?

**Comment by sweatC** [comment]
Fixed twitter meta tag for title: http://jetthoughts-pr-396.surge.sh/

**Comment by ad-sys** [comment]
LGTM, but checked twitter when posted link and no preview for some reason.
Also checked https://cards-dev.twitter.com/ and looks good

File: _includes/head.html [hunk 1]
```diff
@@ -40,7 +39,7 @@
 
   <meta name="twitter:card" content="summary">
   <meta name="twitter:title" content="{{ site.title }}">
-  <meta name="twitter:text:title" content="{{ page.title }}"/>
+  <meta name="twitter:text:title" content="{{ site.title }}"/>
```
Comment (pftg) [inline _includes/head.html]: we need to have 2 titles one per agent inbox page and one per home page. could you support this too?

Comment (pftg) [reply _includes/head.html]: Better to fix in this issue:

- [ ] og:title
- [ ] make different titles for agentinbox case page (find the best fit page title) and for home page

**Review by pftg — CHANGES_REQUESTED** [review]
Check my comments

File: _includes/head.html (hunk 1 above)
Comment (sweatC) [reply _includes/head.html]: @pftg pls provide a title and description for agent inbox. Also, there is a task which will fix it: https://github.com/jetthoughts/jetthoughts.github.io/issues/398

Comment (pftg) [reply _includes/head.html]: @sweatC will be great to see your suggestions, also you could discuss with @andriyParashchuk and @ad-sys what is the best

**Review by andriyParashchuk — COMMENTED** [review]
LGTM

File: _includes/head.html (hunk 1 above)
Comment (andriyParashchuk) [reply _includes/head.html]: Will fix in #398 
@sweatC No need change `page.title`. In #398 we will use `page.title` and `page.deskription` for each page

Comment (sweatC) [reply _includes/head.html]: @andriyParashchuk removed last change.

**Review by pftg — APPROVED** [review]

**Pull request merged**